geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala

```diff
@@ -273,6 +275,41 @@ class AccumuloRasterStore(val connector: Connector,
     }
   }
 
+  def deleteTable(): Unit = {
+    deleteMetaData()
+    deleteProfileTable()
+    deleteRasterTable()
+  }
+
+  private def deleteRasterTable(): Unit = {
+    try {
+      if (tableOps.exists(tableName)) {
+        tableOps.delete(tableName)
+      }
+    }
+  }
+
+  private def deleteProfileTable(): Unit = {
+    try {
+      if (tableOps.exists(profileTable)) {
+        tableOps.delete(profileTable)
+      }
+    }
```

Comment:
logging is not currently setup, should I make the class extend logging or can I raise an exception instead?